### DISCUSSION TITLE
posts/2024/10/eleventy-3-0-debuts/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#55] [Announcements]
# posts/2024/10/eleventy-3-0-debuts/

The latest major version of the best JS-based SSG brings quite a few enhancements.

https://www.brycewray.com/posts/2024/10/eleventy-3-0-debuts/

**Comment by davidsneighbour** [comment]
> the Eleventy project recently became part of Font Awesome. It’s a transition which, one hopes, will firmly bolster the long-term stability of this justly popular SSG.

"I have a bad feeling" about this. Not because it's Font Awesome, which is awesome. But it feels like it was yesterday when Eleventy "joined" (was it) Netlify, which, too, was supposed to bolster long-term stability. That went sideways after just one year with a bitter taste if I remember my lore right.

On it's own feet Eleventy could thrive if there wouldn't be the issue of unpaid developer's work. I don't have any idea of a solution though, so every step is a good one. Even the ones that feel weird.
